spec/forms/teacher_interface/name_and_date_of_birth_form_spec.rb

```diff
@@ -22,16 +22,26 @@
     let(:family_name) { "" }
     let(:date_of_birth) { "" }
 
-    it { is_expected.to be true }
+    it { is_expected.to be false }
```

Comment:
I think we should group this assertion with the `let`s above it in to a `context` for when the fields are blank so it's easier to read.

I also think this test could do with refactoring to use `be_valid` as with the other form tests.